apps/api/src/app/shared/framework/swagger/swagger.controller.ts

```diff
@@ -21,37 +21,37 @@ const options = new DocumentBuilder()
   .addTag(
     'Events',
     `Events represent a change in state of a subscriber. They are used to trigger workflows, and enable you to send notifications to subscribers based on their actions.`,
-    { url: 'https://docs.novu.co/workflows' }
+    { url: 'https://docs.novu.co/workflows?utm_campaign=inapp-swagger' }
```

Comment:
hey @justnems , these `url` fields are intended to be machine-readable, and it is non-standard practice to include `utm` query parameters on these because they can cause issues when machines parse the url for use in their machine applications.

One example, machine parses the `url` and appends the `path` from an OpenAPI operation, the URL becomes:
`POST https://docs.novu.co/workflows?utm_campaign=inapp-swagger/workflows/<workflow-id>`
This is not a valid URI.

Generally, we should avoid adding `utm` attributes to machine generated response for similar machine-readability reasons, and only add them to static pages like `.md` and Web application rendered pages, where the query param itself is hidden behind a hyperlink.

Reply:
Understood. Interestingly, I did find a few examples where that already happened-- there was one where the `utm` query happened mid-URL, and had a path afterward. Clearly not the intention!